components/brave_adaptive_captcha/BUILD.gn

```diff
@@ -0,0 +1,24 @@
+source_set("brave_adaptive_captcha") {
+  sources = [
```

Comment:
this should assert that brave adaptive captcha is enabled

Reply:
Fixed.